src/less-style/structure.less

```diff
@@ -37,6 +37,23 @@ body.vellum-full-screen {
 
                 }
 
+                .fd-commander {
+                    display: flex;
+                    padding: 6px 4px 6px 12px;
+                    background-color: @cc-dark-cool-accent-mid;
+
+                    .fd-add-question-toggle {
+                        color: white;
+                        height: 17px;
+                    }
+
+                    input {
+                        border: 1px solid @cc-dark-cool-accent-low;
+                        flex: 2;
+                        margin: -2px 0;
```

Comment:
It looks like you're using flex and the negative margin so the input takes up all width except the plus icon, and so it lines up nicely with the plus, is that right? And the plus is there so that you can click on it to get out of the commander?

Another option could be to hide the plus icon when the commander is shown, let the input take up the full width, and have a different way to exit the commander. Maybe just an x, something like below. It would mean two clicks to add a question the traditional way (click the x, then click the new-visible add question button) but I'm not certain that clicking the plus is an intuitive way to close the commander in the first place.

<img width="388" alt="screen shot 2017-08-22 at 10 25 28 am" src="https://user-images.githubusercontent.com/1486591/29570680-2205115a-8725-11e7-9e7f-89a3669731a4.png">

<img width="411" alt="screen shot 2017-08-22 at 10 28 48 am" src="https://user-images.githubusercontent.com/1486591/29570687-2725bf72-8725-11e7-997e-c71acc305de5.png">